curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfb6250eacea3f48c6300b2.md

```diff
@@ -20,37 +20,35 @@ And its `alt` attribute value to:
 There should be an `img` element right after the closing `</ul>` tag.
 
 ```js
-assert.equal(document.querySelectorAll('section')?.[1]?.lastElementChild?.nodeName, 'IMG');
+assert.equal(document.querySelectorAll('section')[1].querySelector('ul').nextElementSibling?.nodeName, 'IMG');
 ```
 
 The new image does not have an `alt` attribute. Check that there is a space after the opening tag's name and/or there are spaces before all attribute names.
 
 ```js
-assert.isTrue(document.querySelectorAll('section')?.[1]?.lastElementChild?.hasAttribute('alt'));
+assert.isTrue(document.querySelectorAll('section')[1].querySelector('ul').nextElementSibling?.hasAttribute('alt'));
 ```
 
 The new image should have an `alt` value of `A slice of lasagna on a plate.` Make sure the `alt` attribute's value is surrounded with quotation marks.
 
 ```js
-assert(
-  document.querySelectorAll('section')?.[1]
-    ?.lastElementChild?.getAttribute('alt')
-    .replace(/\s+/g, ' ')
-    .match(/^A slice of lasagna on a plate\.?$/i)
+assert.match(
+  document.querySelectorAll('section')[1].querySelector('ul').nextElementSibling?.getAttribute('alt')
```

Comment:
```suggestion
  document.querySelectorAll('section')[1].querySelector('ul').nextElementSibling?.getAttribute('alt')?
```
getAttribute returns null if the attribute is not found. I don't know if it can work like this or it needs to go on a single line